src/components/pages/myPage/ReservationListCard.tsx

```diff
@@ -0,0 +1,116 @@
+'use client';
+import { Badge } from '@/components/ui/badge';
+import { Button } from '@/components/ui/button';
+import {
+  Card,
+  CardContent,
+  CardDescription,
+  CardHeader,
+  CardTitle,
+  CardFooter,
+} from '@/components/ui/card';
+import { reservationStatus } from '@/lib/constants/reservation';
+import { toCardDate } from '@/lib/utils/dateUtils';
+import { ReservationStatus } from '@/types/activities.type';
+import { Reservation } from '@/types/myReservation.type';
+import Image from 'next/image';
+import { useState } from 'react';
+
+interface MyExperienceCardProps {
+  data: Reservation;
```

Comment:
data 라는 이름은 (사실 우리가 할당,호출,반환 하는 모든게 data니까) 어떤것도 설명해주지 못해서 이름으로서 역할을 못한다고 생각합니다. (때때로 안중요한 상황에서 성의없이 작성하는 떄 '뭔가'라는 의미에서 사용하긴 하지만요)
정체성을 드러낼 수 있을 때는 명확히 작성하시는 걸 습관들이길 추천합니다
```suggestion
  reservation: Reservation;
```